msteams-platform/concepts/design/enable-app-customization.md

```diff
@@ -29,27 +32,25 @@ You can't test this feature during development. App customization isn't supporte
 
```

Comment:
About `Test your app` section: 
Instead of making it a heading, considering making an admonition. May be, important?
Avoid heading that contain only one sentence of content.

Reply:
corrected as per suggestion